src/commands/operator/recover.ts

```diff
@@ -17,12 +17,20 @@ import ApiService from '../../services/api-service'
 import {CrossChainTransaction, Logger, TransactionStatus} from '../../types/api'
 import color from '@oclif/color'
 import {decodeAvailableOperatorJobEvent, decodeLzPacketEvent} from '../../events/events'
+import * as fs from 'fs-extra'
 
 enum Step {
   OPERATOR,
   MESSAGE,
 }
 
+interface IncompleteJobs {
+  // eslint-disable-next-line camelcase
```

Comment:
just curious why the reason for snake case in this particular block?

Reply:
@alexanderattar yea im getting this object from the DB that is in snake case. So out of convenience i made this snake case.